### ISSUE TITLE
rune essence count not updating properly

### ISSUE THREAD

**Issue by Seamooo** [MelvorIdle/melvoridle.github.io#101]
**Describe the bug**
rune essence count does not update in runecrafting tab (while mining rune essence) unless refreshing by switching from tabs or creating a rune

**To Reproduce**
1. go to mining tab and click on rune essence
2. go to runecrafting tab and select a rune
3. note that the value of you have: <rune essence count> does not change even though you are mining

**Expected behavior**
The rune essence count would reactively update as you mine rune essence

**Screenshots**
Not applicable

**Browser**
Google Chrome   79.0.3945.130 (Official Build) (64-bit) (cohort: Stable)

**Console output**
```
www.googletagmanager.com/gtag/js?id=UA-129674422-1:1 Failed to load resource: net::ERR_BLOCKED_BY_CLIENT
2pagead2.googlesyndication.com/pagead/js/adsbygoogle.js:1 Failed to load resource: net::ERR_BLOCKED_BY_CLIENT
save.js?200:45 Loading Save Game...
save.js?200:212 Save Game Loaded...
mastery.js?200:200 Updating Masteries
mastery.js?200:36 Loading Masteries...
smithing2.js?200:217 Loading Smithing...
main.js?203:1022 Game Loaded
cloud.js?203:50 Melvor Cloud Connection: Success
save.js?200:45 Loading Save Game...
save.js?200:212 Save Game Loaded...
save.js?200:228 Save Game Deleted
mastery.js?200:200 Updating Masteries
mastery.js?200:36 Loading Masteries...
smithing2.js?200:217 Loading Smithing...
main.js?203:1022 Game Loaded
cloud.js?203:149 Uncaught ReferenceError: updateOffline is not defined
    at loadCloudSave (cloud.js?203:149)
    at HTMLButtonElement.onclick (index.php:6168)
1.png:1 Failed to load resource: the server responded with a status of 404 ()
3.png:1 Failed to load resource: the server responded with a status of 404 ()
2.png:1 Failed to load resource: the server responded with a status of 404 ()
0.png:1 Failed to load resource: the server responded with a status of 404 ()
4.png:1 Failed to load resource: the server responded with a status of 404 ()
11cloud.js?203:167 Cloud synced
rockHitting.js?200:173 Rock check
cloud.js?203:167 Cloud synced
2rockHitting.js?200:173 Rock check
cloud.js?203:167 Cloud synced
```

**Comment by MelvorIdle** [comment]
I'll have a fix out for next update :)

**Comment by MelvorIdle** [comment]
Confirmed fixed in v0.13
